src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -116,15 +119,25 @@ public static function collectDeprecations($outputFile)
         });
     }
 
+    private function callPhpUnitErrorHandler($type, $msg, $file, $line, $context)
+    {
+        $ErrorHandler = self::$utilPrefix.'ErrorHandler';
+        if (self::$isHandlerInvokable) {
+            $object = new $ErrorHandler($convertDeprecationsToExceptions = true, $convertErrorsToExceptions = true, $convertNoticesToExceptions = true, $convertWarningsToExceptions = true);
```

Comment:
Not sure how to deal with these booleans. In [TestResult class](https://github.com/sebastianbergmann/phpunit/blob/8.3.2/src/Framework/TestResult.php#L631-L636) they are all true by default